### PR TITLE
Player 4169

### PR DESCRIPTION
**Pull request by dallen42**
Don't merge until html5-common is updated with the new constants

### CODE REVIEW

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -39,6 +39,14 @@ require("../../../html5-common/js/utils/environment.js");
           list.push(OO.VIDEO.ENCODING.MP4);
         }
 
+        if (!!videoElement.canPlayType("audio/ogg")) {
```
Comment (pilievOoyala) [inline src/main/js/main_html5.js]: I'm just double checking, but you are doing the !! because 'probably' and 'maybe' are okay but empty string ('') isn't.

Comment (pilievOoyala) [reply src/main/js/main_html5.js]: Nevermind, I see that this is the pattern for all of them.

**Comment by pilievOoyala** [comment]
LGTM

**Comment by aeng7** [comment]
LGTM, but I think html5-common needs to be updated after the merge from yesterday. I'll approve and merge afterwards.

**Pull request merged**